.github/workflows/ci.yml

```diff
@@ -121,6 +121,8 @@ jobs:
           platforms: arm64,ppc64le
       - name: Build and Test Docker Image
         run: core/docker/build.sh
+      - name: Build and Test Docker Image with JDK 22-ea
+        run: core/docker/build.sh -j 22 -e
```

Comment:
The container tests we run are very basic smoke tests, but building the image is expensive. Is it really worth running in every PR? Even with all the recent improvements in test run times, we can still easily create a backlog in the CI.

Reply:
The purpose of this test is to build a docker image and check whether it starts